Uchu.StandardScripts/Equipment/FactionEquipment.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Objects;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.Equipment
+{
+    //lol
+    [ScriptName("assemblyengineer1.lua")]
+    [ScriptName("assemblyengineer2.lua")]
+    [ScriptName("assemblyengineer3.lua")]
+    [ScriptName("assemblyinventor1.lua")]
+    [ScriptName("assemblyinventor2.lua")]
+    [ScriptName("assemblyinventor3.lua")]
+    [ScriptName("assemblysummoner1.lua")]
+    [ScriptName("assemblysummoner2.lua")]
+    [ScriptName("assemblysummoner3.lua")]
+    [ScriptName("sentinelknight1.lua")]
+    [ScriptName("sentinelknight2.lua")]
+    [ScriptName("sentinelknight3.lua")]
+    [ScriptName("sentinelspaceranger1.lua")]
+    [ScriptName("sentinelspaceranger2.lua")]
+    [ScriptName("sentinelspaceranger3.lua")]
+    [ScriptName("sentinelsamurai1.lua")]
+    [ScriptName("sentinelsamurai2.lua")]
+    [ScriptName("sentinelsamurai3.lua")]
+    public class FactionEquipment : SkillSetTriggerTemplate
+    {
+        private bool Started = false;
+
+        //save memory by having this hardcoded nonsense static
+        private static Dictionary<int, (int skillID, int itemsRequired, float cooldownTime, bool method)> sets = new Dictionary<int, (int skillID, int itemsRequired, float cooldownTime, bool method)>
```

Comment:
just FYI, you can simply write `new()` instead of `new Dictionary<int, (int skillID, int itemsRequired, float cooldownTime, bool method)>` here because the type was specified already